src/components/atoms/OcSelect/OcSelect.vue

```diff
@@ -154,6 +154,9 @@ export default {
     &__selected {
       color: var(--oc-color-input-text-default);
     }
+    &__selected {
+      background-color: var(--oc-color-background-default);
```

Comment:
Can you add the introduced `background-color` two lines above into the existing `&__selected`, immediately below `color: ...`?

Reply:
but this also applies to &__search and this doesn't look good

Reply:
sorry, didn't expand the diff on github and didn't see the `&__search` selector